src/sections/Projects/Project-grid/index.js

```diff
@@ -121,6 +121,13 @@ const ProjectPage = () => {
                 </Col>
               </div>
             </Link>
+            <Link
+              to="/cloud-native-management/playground" className="project__card eleven">
+              <div className="project__card-container-eleven">
+                <img src={meshery} alt="Service Mesh Patterns" />
+                <h5>Playground</h5>
```

Comment:
alt is the same as the project title in other project cards, can we do the same here? "Playground" in this case.